services/app-api/forms/naaar.json

```diff
@@ -236,7 +236,7 @@
             "intro": {
               "section": "I. State and program information",
               "subsection": "Provider type coverage",
-              "info": "Instructions will be here for for how to best classify provider types to the categories shown. If your standards include more specific providers, select the core provider type category closest to the provider type and you can provide it as a subcategory when creating standards later in the form."
+              "info": "Instructions will be here for for how to best classify provider types to the categories shown. If your standards apply to more specific provider types, select the most closely aligned provider type category and utilize the subcategory fields available in Section XXXX."
```

Comment:
should "Section XXXX" here be the standards section? I'm guessing we don't want to keep "XXXX" in the report